pandas/tests/test_window.py

```diff
@@ -1135,7 +1135,22 @@ def test_rolling_quantile_series(self):
         s = Series(arr)
         q1 = s.quantile(0.1)
         q2 = s.rolling(100).quantile(0.1).iloc[-1]
+        tm.assert_almost_equal(q1, q2)
```

Comment:
can you move to a separate test and parameterize on the interpolation (inlcude linear as well)

Reply:
these are all coming to the same result value, is that right?